extensions/ql-vscode/src/run-queries-shared.ts

```diff
@@ -198,7 +198,7 @@ export class QueryEvaluationInfo {
         // This could be from the new query server
         // in which case we expect the qlo to be missing so we should ignore it
```

Comment:
This comment is no longer relevant.

Reply:
No it is a still correct. The point was that if we are using the new query server we don't expect a qlo so we shouldn't report that the qlo is missing and report that the DIL is missing. If we are still using the old queryserver then a missing qlo is likely to be the problem.